chronix-server-query-handler/src/main/java/de/qaware/chronix/solr/query/analysis/FunctionValueMap.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright (C) 2016 QAware GmbH
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package de.qaware.chronix.solr.query.analysis;
+
+import de.qaware.chronix.solr.query.analysis.functions.ChronixAggregation;
+import de.qaware.chronix.solr.query.analysis.functions.ChronixAnalysis;
+import de.qaware.chronix.solr.query.analysis.functions.ChronixTransformation;
+
+/**
+ * Simple fixed size map of chronix analysis and value
+ *
+ * @author f.lautenschlager
+ */
+public class FunctionValueMap {
+
+    private final ChronixAnalysis[] analyses;
+    private final ChronixTransformation[] transformations;
+    private final ChronixAggregation[] aggregations;
+
+    private final boolean[] analysisValues;
+    private final double[] aggregationValues;
+
+    private final String[] identifiers;
+
+    private int analysisSize = 0;
```

Comment:
\[Checkstyle\] ERROR: Variable 'analysisSize' explicitly initialized to '0' \(default value for its type\)\.